lib/iris/coords.py

```diff
@@ -625,7 +624,7 @@ def xml_element(self, doc):
         # otherwise.
         if isinstance(self, Coord):
             values_term = "points"
-        elif isinstance(self, Connectivity):
+        elif hasattr(self, "indices"):
```

Comment:
Ideally, we'd want to change this back, but only once `iris.experimental.ugrid` is pulled in to the core of `iris`.